data/sample_signature_activity.txt

```diff
@@ -1,4 +1,4 @@
ig290pos	Sig291pos	Sig292pos	Sig293pos	Sig294pos	Sig295pos	Sig296pos	Sig297pos	Sig298pos	Sig299pos	Sig300pos	Sig1neg	Sig2neg	Sig3neg	Sig4neg	Sig5neg	Sig6neg	Sig7neg	Sig8neg	Sig9neg	Sig10neg	Sig11neg	Sig12neg	Sig13neg	Sig14neg	Sig15neg	Sig16neg	Sig17neg	Sig18neg	Sig19neg	Sig20neg	Sig21neg	Sig22neg	Sig23neg	Sig24neg	Sig25neg	Sig26neg	Sig27neg	Sig28neg	Sig29neg	Sig30neg	Sig31neg	Sig32neg	Sig33neg	Sig34neg	Sig35neg	Sig36neg	Sig37neg	Sig38neg	Sig39neg	Sig40neg	Sig41neg	Sig42neg	Sig43neg	Sig44neg	Sig45neg	Sig46neg	Sig47neg	Sig48neg	Sig49neg	Sig50neg	Sig51neg	Sig52neg	Sig53neg	Sig54neg	Sig55neg	Sig56neg	Sig57neg	Sig58neg	Sig59neg	Sig60neg	Sig61neg	Sig62neg	Sig63neg	Sig64neg	Sig65neg	Sig66neg	Sig67neg	Sig68neg	Sig69neg	Sig70neg	Sig71neg	Sig72neg	Sig73neg	Sig74neg	Sig75neg	Sig76neg	Sig77neg	Sig78neg	Sig79neg	Sig80neg	Sig81neg	Sig82neg	Sig83neg	Sig84neg	Sig85neg	Sig86neg	Sig87neg	Sig88neg	Sig89neg	Sig90neg	Sig91neg	Sig92neg	Sig93neg	Sig94neg	Sig95neg	Sig96neg	Sig97neg	Sig98neg	Sig99neg	Sig100neg	Sig101neg	Sig102neg	Sig103neg	Sig104neg	Sig105neg	Sig106neg	Sig107neg	Sig108neg	Sig109neg	Sig110neg	Sig111neg	Sig112neg	Sig113neg	Sig114neg	Sig115neg	Sig116neg	Sig117neg	Sig118neg	Sig119neg	Sig120neg	Sig121neg	Sig122neg	Sig123neg	Sig124neg	Sig125neg	Sig126neg	Sig127neg	Sig128neg	Sig129neg	Sig130neg	Sig131neg	Sig132neg	Sig133neg	Sig134neg	Sig135neg	Sig136neg	Sig137neg	Sig138neg	Sig139neg	Sig140neg	Sig141neg	Sig142neg	Sig143neg	Sig144neg	Sig145neg	Sig146neg	Sig147neg	Sig148neg	Sig149neg	Sig150neg	Sig151neg	Sig152neg	Sig153neg	Sig154neg	Sig155neg	Sig156neg	Sig157neg	Sig158neg	Sig159neg	Sig160neg	Sig161neg	Sig162neg	Sig163neg	Sig164neg	Sig165neg	Sig166neg	Sig167neg	Sig168neg	Sig169neg	Sig170neg	Sig171neg	Sig172neg	Sig173neg	Sig174neg	Sig175neg	Sig176neg	Sig177neg	Sig178neg	Sig179neg	Sig180neg	Sig181neg	Sig182neg	Sig183neg	Sig184neg	Sig185neg	Sig186neg	Sig187neg	Sig188neg	Sig189neg	Sig190neg	Sig191neg	Sig192neg	Sig193neg	Sig194neg	Sig195neg	Sig196neg	Sig197neg	Sig198neg	Sig199neg	Sig200neg	Sig201neg	Sig202neg	Sig203neg	Sig204neg	Sig205neg	Sig206neg	Sig207neg	Sig208neg	Sig209neg	Sig210neg	Sig211neg	Sig212neg	Sig213neg	Sig214neg	Sig215neg	Sig216neg	Sig217neg	Sig218neg	Sig219neg	Sig220neg	Sig221neg	Sig222neg	Sig223neg	Sig224neg	Sig225neg	Sig226neg	Sig227neg	Sig228neg	Sig229neg	Sig230neg	Sig231neg	Sig232neg	Sig233neg	Sig234neg	Sig235neg	Sig236neg	Sig237neg	Sig238neg	Sig239neg	Sig240neg	Sig241neg	Sig242neg	Sig243neg	Sig244neg	Sig245neg	Sig246neg	Sig247neg	Sig248neg	Sig249neg	Sig250neg	Sig251neg	Sig252neg	Sig253neg	Sig254neg	Sig255neg	Sig256neg	Sig257neg	Sig258neg	Sig259neg	Sig260neg	Sig261neg	Sig262neg	Sig263neg	Sig264neg	Sig265neg	Sig266neg	Sig267neg	Sig268neg	Sig269neg	Sig270neg	Sig271neg	Sig272neg	Sig273neg	Sig274neg	Sig275neg	Sig276neg	Sig277neg	Sig278neg	Sig279neg	Sig280neg	Sig281neg	Sig282neg	Sig283neg	Sig284neg	Sig285neg	Sig286neg	Sig287neg	Sig288neg	Sig289neg	Sig290neg	Sig291neg	Sig292neg	Sig293neg	Sig294neg	Sig295neg	Sig296neg	Sig297neg	Sig298neg	Sig299neg	Sig300neg
```

Comment:
So were these padding zeroes added manually (with a Python script or something else)?

Reply:
Yes, I wrote a 4-line python script that generated the new row, then replaced the original row with it. Same idea with the other data file, but I used `cut/paste` command to replace the column.